src/components/TaskCommandList.tsx

```diff
@@ -48,6 +48,32 @@ class TaskCommandList extends React.Component<Props> {
     return <div>{commandComponents}</div>;
   }
 
+  buildKey(command: ItemOfArray<TaskCommandList_task['commands']>, name) {
+    /**
+     * Returned array indexes guide:
+     * 0: Display name
+     * 1: Material UI icon
+     */
+    // stringbuilder for name to be displayed
+    let sb = []
+    switch (command.type) {
+      case CLONE:
```

Comment:
@fkorotkov I need some help - how can I access the enum for TaskCommandType in the graphql schema?  I put these placeholders there but I need a way to run actual comparisons